pkgs/development/compilers/ghc/head.nix

```diff
@@ -105,7 +105,7 @@ in stdenv.mkDerivation (rec {
     "RANLIB=${stdenv.binutilsCross}/bin/${cross.config}-ranlib"
     "--target=${cross.config}"
     "--enable-bootstrap-with-devel-snapshot"
-  ];
+  ] ++ (lib.optional ("${cross.config}" == "aarch64-apple-darwin14") "--disable-large-address-space");
```

Comment:
It's a minor detail, but I tend to prefer using `stdenv.lib` over expecting `lib` as an explicit argument. Makes no difference in practice, though, just my 2 cents.